### DISCUSSION TITLE
How to add / change main menu?

### DISCUSSION THREAD

**Discussion by spicter** [Budibase/budibase#3954] [Help]
Hi all,

I'm just trying to add main menu items and assign routes to them.

Did not find any documentation on it.

Thx for your help

**Comment by kovrom** [comment]
Navigate to your app. Then: Design -> Layouts -> Navigation Layout. On the right side you'll find Settings panel, click Configure Links. Adjust/Remove/Add new links in the popup window. First column is your link Text, second column is a drop-down with available paths. Click Save.

**Reply by AnotherITcorp** [reply]
Hit there, what if I want to make dropdown menu with child elements ?

**Reply by allanfann** [reply]
I have the same question, how to create multi-layer menu in navigation bar?
